src/controllers/commentController.js

```diff
@@ -85,6 +90,28 @@ const replyComment = async (request, response) => {
   }
 };
 
+const likeComment = async (request, response) => {
+  const { user: { id: userId, isSubscribed }, params: { commentId } } = request;
+  try {
+    if (!isSubscribed) {
+      return responseMessage(response, 403, { error: 'you have to be subscribed to like a comment' });
+    }
+    const commentExist = await findComment(commentId);
+    if (!commentExist) {
+      return responseMessage(response, 404, { error: 'comment does not exist' });
+    }
+    const likeExist = await findCommentLike(userId, commentId);
+    if (likeExist) {
```

Comment:
toggling a comment like may appear to be unpredictable in the future, I suggest there is a parameter like { like } or { dislike } so that the system can tell with confidence what the user's intention is

Reply:
yeah, that is a good point but I feel it makes no difference. You either want to like or unlike a comment so anytime you hit the endpoint it takes action based on your previous interaction with that endpoint or the lack of it

Reply:
ok